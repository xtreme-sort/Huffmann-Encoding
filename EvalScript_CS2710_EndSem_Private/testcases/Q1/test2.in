BuildFrequencyTable
qFqsGJPcSpUgIWFaW5G_MAHZejdzj3xRtdfGXR1LUaO3pxJQhxmTWTMJZVWJzPVIwUv_IPgE7GdZ69YNAowoxNFOq6vC7X35FWRaUKGlThb5Gd81gP_KwQC6JlK0zg1Uk6BA_YK7YiLS15lwb2ctX0m5k2Pfe3jGWlq7lrpdc508TzeC_Ia3NUhkbKj37ZCrTN0QWJgc1umTUY1I50hAY3sCkmLnOA7Z6uRz5VLgKvHU0ikCeUojHUEzkq_ZGqBlhmzmRajXknM6m86sOXw7MItmHb4l9BD6tBhMWB1mtdlX1VKjr1FIrD0JxF7g09KS7rmszTxaSwt4dsypvOyad3sN0vpQU2rTr_oMPQVJs6w5Arq_0wcM89RVXNiiKabSG4rPRuRcrCPqPJMFBPFPcdntvyvoSqjoX0OVBF067pVltKYvZELg7Noc24nIVF6M6TQ1tRO0iyqSTynixdN9OOTix_egZv9sObtt81Vpvsxs6PTSpLZilQMVZ1sXbvTaUxLAjsK0NC_0GBCJgJsAcAe6QywJmFxbGYomL0yTJtqIZLzQskX4kaEsSo7QOwUxEhkJLUyS7e_tF1aj3K5m7XaKLRDWnS1Iml7mU67Ssi1HM5Zb87w8Jie9qlC8i9EXBV3q5h9vjXvqZT1U30k0eUA5Kje_bnOHjbneFClDpqs6yexwZORYVpC9NusYpLkiKGeRv6toaDJxAEI11hzCvK5oo3U6OnyNAbF3fEk0Gilad5gjBBPmSbB_QG5Ad5Y6O5UlAsVF9p3gApNZEhvzHv3C7J8WM0CfLgMwK5XN259EPAoigCwJY5rglfrMh9kQZBKS4uAsA
Quit
