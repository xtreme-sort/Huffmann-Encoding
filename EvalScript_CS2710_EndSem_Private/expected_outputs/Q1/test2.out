0 20
1 18
2 5
3 15
4 6
5 21
6 19
7 18
8 9
9 13
A 18
B 13
C 16
D 5
E 10
F 15
G 14
H 7
I 11
J 18
K 18
L 13
M 14
N 13
O 15
P 14
Q 12
R 12
S 14
T 15
U 19
V 14
W 10
X 14
Y 11
Z 16
_ 13
a 14
b 13
c 9
d 12
e 13
f 5
g 15
h 11
i 14
j 14
k 14
l 16
m 16
n 9
o 13
p 13
q 15
r 12
s 20
t 14
u 5
v 17
w 14
x 14
y 10
z 11
